content/md/franklin-insights/franklin-key-figures.md

```diff
@@ -0,0 +1,38 @@
+---
+id: franklin-key-figures
+themes: franklin-insights
+title: Franklin **key figures**
+popular: false
+ee-only: true
+related: franklin-insights-subscriptions
+---
+# A dedicated dashboard
+You want to follow your activity using Franklin? There is a dashboard for this!
+1. Click on `Activity`
+2. Click on `Franklin key figures`
+
+![Franklin's dashboard screen](../img/Akeneo_Screenshot_Franklin_Insights_key-figures.png)
+
+# Quality Highlights KPI
+
+The first section of the dashboard lets you know the number of product values Franklin added to your catalog. There are several ways for Franklin to contribute to your catalog enrichment:
+- Create a new attribute
+- Add an attribute value
+- Detect an error in an attribute value
+
+
+To help you follow Franklin's activity, you can follow the following key figures:
+- `Attribute created`: attributes suggested and created using the attribute mapping screen,
+- `Attribute added to family`: attributes created and added using the attribute mapping screen,
+- `Attribute values validated`: attribute values for which Franklin found the same value as the one filled in the PIM,
+- `Attribute values in error`: errors detected in the PIM's attribute values,
+- `Attribute values suggested`: attribute content suggested by Franklin and validated in the PIM,
```

Comment:
Not clear, what do you mean by validated in the PIM?
I think this one is simply "attribute value suggested by Franklin", isn't it?